packages/smart-wallet/package.json

```
@@ -17,9 +17,11 @@
   },
   "devDependencies": {
     "@agoric/cosmic-proto": "^0.3.0",
+    "@endo/bundle-source": "^2.5.2",
```

Is there any chance this would be compatible with 2.5.1 to avoid having duplicate deps?
```suggestion
    "@endo/bundle-source": "^2.5.1",
```

(you'll need to back out the `yarn.lock` change and land it again)